pipeline.py

```diff
@@ -535,6 +531,54 @@ def generate_pipeline(mock):
             compiler.Compiler().compile(pipeline_func, pipeline_file)
 
 
+@cli.command(name="run")
+@click.option(
+    "--mock",
+    type=click.Choice(MOCKED_STAGES, case_sensitive=False),
+    help="Mock part of the pipeline",
+    multiple=True,
+    default=[],
+)
+@click.option("-e", "--experiment", help="Set KFP experiment name.")
+@click.option("-r", "--run", "run_name", help="Set KFP run name.")
+@click.option(
+    "-p",
+    "--param",
+    help="Override default parameters in KEY=VALUE format. Default parameters are suitable for dev cluster - the MOC cluster, `ilab` namespace.",
+    multiple=True,
+)
+def run(mock, experiment, run_name, param):
+    """
+    Run the pipeline immediately against current kubernetes context (cluster and namespace).
+
+    Command sets expected dev-cluster friendly default values when submitting.
+    """
+    from utils.kfp_client import get_kfp_client
+
+    client = get_kfp_client()
+
+    dev_arguments = {
+        "k8s_storage_class_name": "nfs-csi",
+        "sdg_base_model": "s3://ilab-pipeline-b1d4c2b1-ab00-4e7f-b985-697bda3df385/instructlab-base-importer/648f36d0-e3f0-43b8-8adb-530576beb675/ilab-importer-op/model/granite-7b-starter",
+        "train_nproc_per_node": 2,
+    }
```

Comment:
Why are these hardcoded if we now have `--param`?

Reply:
This is here to simplify our dev workflow. You can always override them with `--param`.